server/services/teamsSummaryCountService.ts

```diff
@@ -198,7 +226,22 @@ export default class TeamsSummaryCountService {
         { critical: 0, high: 0 },
       )
 
+      const totalSummaryCount = summaryFixedCritical + summaryFixedHigh + summaryUnfixedCritical + summaryUnfixedHigh
+      const totalScanResultCount = counts.critical + counts.high
+
+      logger.info(
```

Comment:
Can you remove all this logging? Put it in if you need it at a later date. But remember these are going into appInsights and can make it more difficult to get to the useful stuff.